packages/nimble-components/src/theme-provider/design-tokens.ts

```diff
@@ -306,6 +306,18 @@ export const iconSize = DesignToken.create<string>(
 export const drawerWidth = DesignToken.create<string>(
     styleNameFromTokenName(tokenNames.drawerWidth)
 ).withDefault('784px');
+export const dialogSmallWidth = DesignToken.create<string>(
+    styleNameFromTokenName(tokenNames.dialogSmallWidth)
+).withDefault('400px');
+export const dialogSmallMaxHeight = DesignToken.create<string>(
+    styleNameFromTokenName(tokenNames.dialogSmallMaxHeight)
+).withDefault('600px');
+export const dialogLargeWidth = DesignToken.create<string>(
+    styleNameFromTokenName(tokenNames.dialogLargeWidth)
+).withDefault('1024px');
+export const dialogLargeHeight = DesignToken.create<string>(
+    styleNameFromTokenName(tokenNames.dialogLargeHeight)
+).withDefault('680px');
```

Comment:
Where are these values coming from?

Reply:
- The width and the max height for the small ones are exactly the same ones as those that are [hardcoded now](https://github.com/ni/nimble/blob/43176dfe4924fd4a40af5d014fee7961b0022ad9/packages/nimble-components/src/dialog/styles.ts#L34C1-L35C27). I followed @jattasNI 's indication of creating tokens for this one, as he mentioned in the issue that I linked.
- The width and height for the large component have been decided by Leslie for the software install workflow in SLE. We followed up with our PO, TJ, and the decision was to have these values as a standard for all large dialogs